gen3-client/mocks/mock_configure.go

```diff
@@ -34,23 +34,33 @@ func (m *MockConfigureInterface) EXPECT() *MockConfigureInterfaceMockRecorder {
 	return m.recorder
 }
 
+// GetConfigPath mocks base method
+func (m *MockConfigureInterface) GetConfigPath() (string, error) {
+	ret := m.ctrl.Call(m, "GetConfigPath")
+	ret0, _ := ret[0].(string)
+	ret1, _ := ret[1].(error)
```

Comment:
what does this do?

Reply:
This is how the `gomock` framework works for testing. It genereate mock interfaces for your code using reflections to call the respective functions and get returned values https://github.com/golang/mock/blob/4657bf943413a7dd4d68abea5dcbf06807042949/gomock/call.go#L112

All these files with `mock_` as prefixes in their filenames are actually automatically generated by `mockgen` https://github.com/golang/mock#running-mockgen